### PR TITLE
add delete user by id

### PR DESCRIPTION
**Pull request by Andrij-Maksymyshyn**

### CODE REVIEW

File: api/users/usersRouter.js [hunk 1]
```diff
@@ -2,18 +2,25 @@ const usersRouter = require("express").Router();
...
 
 usersRouter.use("/:userId", objectIdValidator("userId"));
 
 usersRouter.put("/:userId", validate(updateUserSchema), controllers.updateUser);
 
+usersRouter.patch(
+  "/:userId/isDeleted",
+  validate(deleteUserSchema),
+  controllers.updateUser
+);
```
Comment (chestnyh) [inline api/users/usersRouter.js]: You should use DELETE method here.
No need for `isDeleted ` in url in this case

File: api/users/usersRouter.js [hunk 2]
```diff
@@ -2,18 +2,25 @@ const usersRouter = require("express").Router();
...
 usersRouter.post("/", validate(newUserSchema), controllers.createUser);
 
 usersRouter.use("/:userId", objectIdValidator("userId"));
 
 usersRouter.put("/:userId", validate(updateUserSchema), controllers.updateUser);
 
+usersRouter.patch(
+  "/:userId/isDeleted",
+  validate(deleteUserSchema),
+  controllers.updateUser
```
Comment (chestnyh) [inline api/users/usersRouter.js]: Didn't get how you delete user, I see you just updateUser

File: postman/Blog.postman_collection.json [hunk 3]
```diff
@@ -100,6 +100,35 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "Delete user by id",
+			"request": {
+				"method": "PATCH",
```
Comment (chestnyh) [inline postman/Blog.postman_collection.json]: Should be DELETE method

File: api/users/usersRouter.js (hunk 1 above)
Comment (Andrij-Maksymyshyn) [reply api/users/usersRouter.js]: I've fixed problems already. See commit below.

File: api/users/usersRouter.js (hunk 2 above)
Comment (Andrij-Maksymyshyn) [reply api/users/usersRouter.js]: I've fixed problems already. See commit below.

File: postman/Blog.postman_collection.json (hunk 3 above)
Comment (Andrij-Maksymyshyn) [reply postman/Blog.postman_collection.json]: I've fixed problems already. See commit below.

**Review by chestnyh — APPROVED** [review]

**Pull request merged**